papaparse.js

```diff
@@ -57,6 +57,7 @@
 	Papa.BAD_DELIMITERS = ['\r', '\n', '"', Papa.BYTE_ORDER_MARK];
 	Papa.WORKERS_SUPPORTED = !IS_WORKER && !!global.Worker;
 	Papa.SCRIPT_PATH = null;	// Must be set by your code if you use workers and this lib is loaded asynchronously
+	Papa.SCRIPT_PATH_AS_BLOB_URL = false;	// Should be used when you want to use blob urls as 'SCRIPT_PATH' location.
```

Comment:
I think we should always use one method for loading workers. So for me this variable is useless.

Reply:
I agree it would be better with just one method, but the blob url support nowadays is not very high.
http://caniuse.com/#feat=bloburls

Reply:
I see it's supported on all major browsers. So it's ok for my side.